src/quo2/components/navigation/page_nav.cljs

```diff
@@ -179,16 +190,14 @@
      :left-section 
      {:type                  button-type
       :on-press              event
-      :icon                  icon 
-      :icon-color            color
```

Comment:
The docstring was outdated and the parameters specified were ignored, I fixed it and added support for other parameters.